lib/token.ex

```diff
@@ -137,7 +137,7 @@ defmodule AuthPlug.Token do
       # Check for JWT in Headers:
       Enum.count(get_req_header(conn, "authorization")) > 0 ->
         conn.req_headers
-        |> List.keyfind("authorization", 0)
+        |> List.keyfind("authorization", 0, "")
```

Comment:
Return empty string if "authorization" not find in the list

Reply:
Makes sense to have a default. ✅